src/app/home/_components/letter-countdown.tsx

```diff
@@ -0,0 +1,107 @@
+import { Text } from "@/components/common"
+
+type LetterStatus = "EMPTY" | "IN_DELIVERY" | "ARRIVED"
+type TextColor = "secondary" | "tertiary"
+
+interface Letter {
+  id: number
+  scheduleDate: string
+}
+
+interface LetterCountdownProps {
+  letterList: Letter[]
+}
+
+export const WAITING_TEXT: Record<
+  LetterStatus,
+  {
+    subtitle: string | ((daysLeft: number) => string)
+    subtitleColor: TextColor
+    title: string
+  }
+> = {
+  EMPTY: {
+    subtitle: "오고 있는 편지가 없어요",
+    subtitleColor: "tertiary",
+    title: "주소를 공유해 편지를 받아보세요!",
+  },
+  IN_DELIVERY: {
+    subtitle: (daysLeft: number) => `D-${daysLeft}`,
+    subtitleColor: "secondary",
+    title: "열심히 배달 중...",
+  },
+  ARRIVED: {
+    subtitle: "D-Day",
+    subtitleColor: "secondary",
+    title: "편지가 도착했어요!",
+  },
+}
+
+// TODO : 서버에서 데이터 넘겨주는 거 보고 사용하기.
+// 서버에서 남은 날짜 + status 넘겨주면 필요 없고 날짜만 주면 utils로 옮기기
+// 근데 생각해보니까 공통 아니라서 util로 안 옮겨도 될 것 같기도 하고...
+const getLetterStatus = (
+  scheduleDate: string,
+): { status: LetterStatus; daysLeft?: number } => {
+  const today = new Date()
+  const target = new Date(scheduleDate)
+  const diff = Math.ceil(
+    (target.getTime() - today.getTime()) / (1000 * 60 * 60 * 24),
+  )
+
+  if (diff > 0) return { status: "IN_DELIVERY", daysLeft: diff }
+  if (diff === 0) return { status: "ARRIVED" }
+
+  return { status: "EMPTY" }
+}
+
+export const LetterCountdown = ({ letterList }: LetterCountdownProps) => {
+  const { status, daysLeft } = letterList[0]
+    ? getLetterStatus(letterList[0].scheduleDate)
+    : { status: "EMPTY" as const }
+
+  const { title, subtitle, subtitleColor } = WAITING_TEXT[status]
+
+  return (
+    <section className="bg-letter flex h-[378px] w-full flex-col items-center justify-between rounded-3xl bg-blue-50 px-4 pt-7 pb-3">
+      <section className="flex flex-col items-center">
+        <section className="flex flex-col items-center gap-2" 
+            aria-label={`${subtitle} ${title}`}>
+          <Text
+            variant="heading"
+            size="small"
+            font="Ownglyph ryurue"
+            color={subtitleColor}
+            className="leading-[1.375rem] font-normal"
+          >
+            {typeof subtitle === "string" ? subtitle : subtitle(daysLeft ?? 0)}
+          </Text>
+          <Text
+            variant="heading"
+            font="Ownglyph ryurue"
+            className="leading-7 font-normal"
+          >
+            {title}
+          </Text>
+        </section>
```

Comment:
각 텍스트나 요소로 해당 부분을 충분히 설명할 수 있다면, aria의 사용은 가급적 지양하는 게 좋아요!
```suggestion
        <section className="flex flex-col items-center gap-2">
          <Text
            as="h2"
            variant="heading"
            size="small"
            font="Ownglyph ryurue"
            color={subtitleColor}
            className="leading-[1.375rem] font-normal"
          >
            {typeof subtitle === "string" ? subtitle : subtitle(daysLeft ?? 0)}
          </Text>
          <Text
            as="h3"
            variant="heading"
            font="Ownglyph ryurue"
            className="leading-7 font-normal"
          >
            {title}
          </Text>
        </section>
```